qa/tasks/cephfs/test_volume_client.py

```diff
@@ -808,16 +817,28 @@ def test_multitenant_volumes(self):
 
         # Verify that the volume metadata file stores info about auth IDs
         # and their access levels to the volume, versioning details, etc.
-        expected_vol_metadata = {
-            "version": 2,
-            "compat_version": 1,
-            "auths": {
-                "guest": {
-                    "dirty": False,
-                    "access_level": u"rw"
+        if self.py_version == 'python3':
+            expected_vol_metadata = {
+                "version": 2,
+                "compat_version": 1,
+                "auths": {
+                    "guest": {
+                        "dirty": False,
+                        "access_level": "rw"
+                    }
+                }
+            }
+        else:
+            expected_vol_metadata = {
```

Comment:
why do you think in python2, `json.loads()` returns string in `unicode` ?

Reply:
This was rather a test-driven modification. To demonstrate it -

```
$ python2
Python 2.7.14 (default, Mar 14 2018, 16:45:33) 
[GCC 8.0.1 20180222 (Red Hat 8.0.1-0.16)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> json.loads('["foo", {"bar":["baz", null, 1.0, 2]}]')
[u'foo', {u'bar': [u'baz', None, 1.0, 2]}]
>>> 

$ python3
Python 3.6.5 (default, Mar 29 2018, 18:20:46) 
[GCC 8.0.1 20180317 (Red Hat 8.0.1-0.19)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> json.loads('["foo", {"bar":["baz", null, 1.0, 2]}]')
['foo', {'bar': ['baz', None, 1.0, 2]}]
>>> 
```

Reply:
I think the two dicts will still be equal:

````
$ python3
Python 3.6.5 (default, May 11 2018, 04:00:52) 
[GCC 8.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> {u"foo": u"bar"}  == {"foo": "bar"}
True
````

So this change should not be necessary I think.